Source/MQTTnet.AspnetCore/ReaderExtensions.cs

```diff
@@ -81,23 +83,24 @@ static bool TryReadBodyLength(ref ReadOnlySequence<byte> input, out int headerLe
             var index = 1;
             headerLength = 0;
             bodyLength = 0;
-            
-            var temp = input.Slice(0, Math.Min(5, input.Length)).GetMemory().Span;
+
+            var valueSequence = input.Slice(0, Math.Min(5, input.Length));
+            var valueSpan = valueSequence.IsSingleSegment ? valueSequence.First.Span : valueSequence.ToArray();
```

Comment:
```suggestion
            var valueSpan = valueSequence.IsSingleSegment ? valueSequence.FirstSpan : valueSequence.ToArray();
```

Reply:
We cannot use `FirstSpan` Api under some target frameworks, such as `netcoreapp` and `netstandard`